src/apps/external_executor/executor_code_id.h

```diff
@@ -16,6 +16,12 @@ struct ExecutorNodeInfo
   ccf::Attestation attestation;
   std::vector<ccf::NewExecutor::EndpointKey> supported_endpoints;
 };
+using ExecutorId = ccf::EntityId<ccf::NodeIdFormatter>;
+typedef std::map<ExecutorId, ExecutorNodeInfo> ExecutorIDMap;
+typedef std::map<ExecutorId, crypto::Pem> ExecutorCertsMap;
```

Comment:
`using` rather than `typedef` here, for consistency and because it's more modern C++?

Reply:
yes for consistency, wouldn't say it's not modern enough :)